src/instance/minecraft/commands/KuudraCommand.ts

```diff
@@ -0,0 +1,44 @@
+import { ChatCommandContext, ChatCommandHandler } from '../common/ChatInterface'
+import { Client, HypixelSkyblockMemberRaw } from 'hypixel-api-reborn'
+
+enum Kuudra {
+  'basic' = 'none',
+  'hot' = 'hot',
+  'burning' = 'burning',
+  'fiery' = 'fiery',
+  'infernal' = 'infernal'
+}
```

Comment:
Would it be possible to detect:
- insensitive-case 
- "t1"-"t5" 
- 1-5?

Reply:
also done